package.json

```diff
@@ -2,9 +2,10 @@
   "main": "node_modules/expo/AppEntry.js",
   "private": true,
   "dependencies": {
+    "@firebase/firestore": "^0.3.4",
     "axios": "^0.17.1",
     "expo": "24.0.0",
-    "firebase": "^4.9.0",
+    "firebase": "^4.10.1",
```

Comment:
We'll be using these packages for this auth work and also for the database work in GH-26.